### PR TITLE
Fix ci, reduce the scope

### PR DESCRIPTION
**Pull request by Gsantomaggio**

### CODE REVIEW

File: .github/workflows/test.yaml [hunk 1]
```diff
@@ -24,12 +23,14 @@ jobs:
...
         with:
-          poetry-version: ${{ matrix.poetry-version }}
+          version: 1.1.10
+          virtualenvs-create: false
+          virtualenvs-in-project: false
       - name: poetry install
-        run: poetry install
+        run: poetry install --no-root
       - name: isort check-only
         run: isort --check-only .
```
Comment (wonesy) [inline .github/workflows/test.yaml:35]:
```suggestion
        run: poetry run isort --check-only .
```

File: .github/workflows/test.yaml [hunk 2]
```diff
@@ -24,12 +23,14 @@ jobs:
...
       - name: black check
```
Comment (wonesy) [inline .github/workflows/test.yaml:36]: same with black: `poetry run black --check .`

Comment (wonesy) [inline .github/workflows/test.yaml:36]: same with black: `poetry run black --check .`

File: .github/workflows/test.yaml (hunk 1 above)
Comment (wonesy) [reply .github/workflows/test.yaml:35]: seems like `virtualenvs-create: false` isn't working

File: .github/workflows/test.yaml (hunk 2 above)
Comment (Gsantomaggio) [reply .github/workflows/test.yaml:36]: thanks a lot!!!

**Pull request closed without merging**